commit 2b37a26dcec389723f75d69d3da9c2f15f6c317d
Author: Mauro Carvalho Chehab <mchehab+huawei@kernel.org>
Date:   Wed May 26 12:41:27 2021 +0200

    ci.yml: Fix the job for it to run on a single arch
    
    There were some issues on the previous content. Fix them, in
    order to allow it to build on a single architecture.
    
    Signed-off-by: Mauro Carvalho Chehab <mchehab+huawei@kernel.org>

diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 5b3e757..747a844 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1,34 +1,23 @@
 name: CI
 
-# Should run only on branches and PR, as "on_tag.yml" will handle tags
 on:
+  workflow_dispatch:
   push:
-    branches: master test
   pull_request:
-    branches: master
 
 jobs:
-
-#
-# Linux
-#
   Ubuntu:
     name: Ubuntu
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        arch: [x64_64, aarch64, armv7, ppc64le]
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-      with:
-        arch: ${{ matrix.arch }}
-    - name: prepare
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y build-essential sqlite3
-    - name: build
-      run: |
-        autoreconf -vfi
-        ./configure --enable-all
-        make
-        sudo make install
+      - uses: actions/checkout@v2
+      - name: prepare
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential sqlite3
+      - name: build
+        run: |
+          autoreconf -vfi
+          ./configure --enable-all
+          make
+          sudo make install
